ext/server/device_impl.cpp

```diff
@@ -673,7 +673,12 @@ namespace PyDeviceImpl
     		boost::python::object disp_level, bool device_level = false)
     {
 //        PyCmd *py_cmd_ptr = nullptr;
-        Tango::Command *cmd_ptr = nullptr;
+	    #if _MSC_VER < 1600 //MSVC version <8
+	    	#define NULL ((void *)0)
+        	Tango::Command *cmd_ptr = (struct Tango::Command *) malloc(sizeof(struct Tango::Command));
```

Comment:
This is weird.  Why would we need to allocate memory for one special version of compiler?  Also, the definition of `NULL` on the previous line hasn't been used.

Reply:
Agree. Looks like a memory leak to me.
Do you remember why you did this @NexeyaSGara ?